src/Uno.UI.Runtime.Skia.Wpf/Extensions/UI/Xaml/Controls/TextBoxViewExtension.cs

```diff
@@ -160,7 +164,19 @@ public void SetIsPassword(bool isPassword)
 			// No support for now.
 		}
 
-		public void Select(int start, int length) => _currentInputWidget?.Select(start, length);
+		public void Select(int start, int length)
+		{
+			_currentInputWidget?.Select(start, length);
+
+#if __SKIA__
+			if (_currentInputWidget == null)
+			{
+				this.StartEntry();
+
+				_currentInputWidget?.Select(start, length);
```

Comment:
`StartEntry()` call should guarantee a non-null `_currentInputWidget` I think.

```suggestion
				_currentInputWidget!.Select(start, length);
```